### PR TITLE
abis: Split off EPOLL_CLOEXEC into it's own abi header

### PR DESCRIPTION
**Pull request by Dennisbonke**
Split off from #541 

Part of the mlibc LFS project.

### CODE REVIEW

File: options/linux/include/sys/epoll.h [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 #include <stdint.h>
 #include <abi-bits/signal.h>
+#include <abi-bits/epoll.h>
 
-#define EPOLL_CLOEXEC 1
 #define EPOLL_NONBLOCK 2
```
Comment (avdgrinten) [inline options/linux/include/sys/epoll.h:8]: Shouldn't we also move `EPOLL_NONBLOCK` at the same time?

Comment (Dennisbonke) [reply options/linux/include/sys/epoll.h:8]: I'm unsure, I don't see `EPOLL_NONBLOCK` on my host so I can't tell if the value is correct.

Comment (avdgrinten) [reply options/linux/include/sys/epoll.h:8]: Seems to be a glibc artifact: https://sourceware.org/bugzilla/show_bug.cgi?id=14985
We should probably remove it entirely.

Comment (Dennisbonke) [reply options/linux/include/sys/epoll.h:8]: Will do

**Comment by 64** [comment]
Conflicts.

**Review by 64 — APPROVED** [review]
LGTM

**Pull request merged**